x/batching/module.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
Is there a reason we're using this directly instead of the Cosmos telemetry package? (which wraps this package as far as I can see).

Reply:
Which also makes sense to me? There's two different metrics endpoints for them entirely. Telemetry enables a `/metrics` on the API side. While prometheus has it's own separate port and metrics endpoint.

Reply:
Then were are our current metrics coming from? :P I don't see any calls to this package directly in the SDK code

Reply:
Aha ok we can get it to work via telemetry, but it requires in the node's `app.toml` under the `telemetry` section:

- `prometheus-retention-time` to be greater than 0
- and `metrics-sink = "prometheus` 

And you still need to enable prometheus to get them to prometheus.